web/src/components/datasets/DatasetInfo.tsx

```diff
@@ -95,44 +127,97 @@ const DatasetInfo: FunctionComponent<DatasetInfoProps> = (props) => {
         />
       )}
       {datasetFields.length > 0 && (
-        <Table size='small'>
-          <TableHead>
-            <TableRow>
-              <TableCell align='left'>
-                <MqText subheading inline>
-                  {i18next.t('dataset_info_columns.name')}
-                </MqText>
-              </TableCell>
-              <TableCell align='left'>
-                <MqText subheading inline>
-                  {i18next.t('dataset_info_columns.type')}
-                </MqText>
-              </TableCell>
-              <TableCell align='left'>
-                <MqText subheading inline>
-                  {i18next.t('dataset_info_columns.description')}
+        <>
+          <Table size='small'>
+            <TableHead>
+              <TableRow>
+                <TableCell align='left'>
+                  <MqText subheading inline>
+                    {i18next.t('dataset_info_columns.name')}
+                  </MqText>
+                </TableCell>
+                <TableCell align='left'>
+                  <MqText subheading inline>
+                    {i18next.t('dataset_info_columns.type')}
+                  </MqText>
+                </TableCell>
+                <TableCell align='left'>
+                  <MqText subheading inline>
+                    {i18next.t('dataset_info_columns.description')}
+                  </MqText>
+                </TableCell>
+                <TableCell align='left'></TableCell>
+              </TableRow>
+            </TableHead>
+            <TableBody>
+              {datasetFields.map((field) => {
+                return (
+                  <TableRow key={field.name}>
+                    <TableCell align='left'>{field.name}</TableCell>
+                    <TableCell align='left'>{field.type}</TableCell>
+                    <TableCell align='left'>{field.description || 'no description'}</TableCell>
+                    <TableCell>
+                      <ReadMoreIcon
+                        onClick={() => handleOpen(field.name)}
+                        sx={{ align: 'Right' }}
+                      ></ReadMoreIcon>
+                    </TableCell>
+                  </TableRow>
+                )
+              })}
+            </TableBody>
+          </Table>
+          <Drawer
+            elevation={0}
+            anchor='right'
+            open={open}
+            onClose={handleClose}
+            sx={{ zIndex: 1300 }}
```

Comment:
Can you reference a variable for this in the theme?

Reply:
yeah cool will move this out.

Reply:
Cool will fix this up.